src/main/java/org/broadinstitute/hellbender/tools/copynumber/CreateReadCountPanelOfNormals.java

```diff
@@ -0,0 +1,262 @@
+import org.apache.spark.api.java.JavaSparkContext;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hdf5.HDF5Library;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.CopyNumberProgramGroup;
+import org.broadinstitute.hellbender.engine.spark.SparkCommandLineProgram;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.copynumber.annotation.GCBiasCorrector;
+import org.broadinstitute.hellbender.tools.copynumber.coverage.denoising.svd.HDF5SVDReadCountPanelOfNormals;
+import org.broadinstitute.hellbender.tools.copynumber.coverage.readcount.SimpleCountCollection;
+import org.broadinstitute.hellbender.tools.copynumber.formats.CopyNumberStandardArgument;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.ListIterator;
+import java.util.stream.Collectors;
+
+/**
+ * Tool to create a panel of normals (PoN) given read counts for control samples.  These should be either
+ * HDF5 or TSV output files generated by {@link CollectFragmentCounts}.
+ *
+ * <p>
+ *     The input read counts are first transformed to log2 fractional coverages and preprocessed
+ *     according to specified filtering and imputation parameters.  Singular value decomposition (SVD)
+ *     is then performed to find the first {@code numberOfEigensamples} principal components,
+ *     which are stored in the PoN.  Some or all of these principal components can then be used for
+ *     denoising case samples with {@link DenoiseReadCounts}; it is assumed that the principal components used
+ *     represent systematic sequencing biases (rather than statistical noise).  Examining the singular values,
+ *     which are also stored in the PoN, may be useful in determining the appropriate number
+ *     of principal components to use for denoising.
+ * </p>
+ *
+ * <p>
+ *     If annotated intervals (produced by {@link AnnotateIntervals} are provided, explicit GC-bias correction
+ *     will be performed by {@link GCBiasCorrector} before filtering and SVD.  GC-content information for the
+ *     intervals will be stored in the PoN and used to perform explicit GC-bias correction identically when denoising
+ *     case samples.  Note that if annotated intervals are not provided, it is still likely that GC-bias correction
+ *     is implicitly performed by the SVD denoising (i.e., some of the principal components arise from GC bias).
+ * </p>
+ *
+ * <h3>Examples</h3>
+ *
+ * <pre>
+ * gatk-launch --javaOptions "-Xmx4g" CreateReadCountPanelOfNormals \
```

Comment:
Typically I just use a bash for loop, but I agree that this could be nice.  However, this seems like something we should standardize across all tools that accept multiple `-I`, rather than just for this tool.  Is this something that we already do?  If not, maybe @vdauwera or @cmnbroad have some thoughts?